tutorials/SpectroscopicDataReductionBasics/2-WavelengthCalibration.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Write a sentence introducing the purpose of the next three blocks of code. Looks like you are filtering out regions of the spectrum that look bad.    Need to say that explicitly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>